File: RedisAPI/Models/Vlasnik.cs
using Redis.OM.Modeling;

namespace Redis.OM.Skeleton.Model;


[Document(StorageType = StorageType.Json, Prefixes = new []{"Vlasnik"})]
public class Vlasnik
{
    [RedisIdField] [Indexed]public string Id { get; set; }

    [Indexed] public string Prezime {get; set;} = String.Empty;
    
    [Indexed] public string Email {get; set;} = String.Empty;

    [Indexed] public string Brtelefona {get; set;} = String.Empty;

    [Indexed] public List<string> ObjektiId {get; set;}

    [Indexed] public string Ime {get; set;} = String.Empty;

    [Indexed] public string Userime {get; set;} = String.Empty;

    [Indexed] public string Passwordhash {get; set;} = String.Empty;

    [Indexed] public string Passwordsalt {get; set;} = String.Empty;
}

File: RedisAPI/Client/Vlasnik.js
export class Vlasnik {
    constructor(id, Prezime, Email, BrTelefona, ObjekatId, Ime, Userime, jwt) {
        this.id = id;
        this.prezime = Prezime;
        this.email = Email;
        this.brTelefona = BrTelefona;
        this.objekatId = ObjekatId;
        this.ime = Ime;
        this.userime = Userime;
        this.jwt = jwt;
    }
}